client/webpack.config.js

```diff
@@ -53,15 +53,16 @@ const extraPath = browserslistEnv === 'defaults' ? 'fallback' : browserslistEnv;
 const cachePath = path.resolve( '.cache', extraPath );
 const shouldUsePersistentCache = process.env.PERSISTENT_CACHE === 'true';
 
+console.log( 'Client build started...' );
+
 // Readonly cache prevents writing to the cache directory, which is good for performance.
 // However, on trunk (and when generating cache images), we want to write to the cache
 // so that we can then update the cache to use in subsequent builds. While this costs
 // a minute in the current build, an updated cache saves 2 minutes in many future builds.
 // Note that in local builds, IS_DEFAULT_BRANCH is not set, in which case we should also write to the cache.
 const shouldUseReadonlyCache = ! (
-	process.env.IS_DEFAULT_BRANCH === 'true' ||
-	process.env.GENERATE_CACHE_IMAGE === 'true' ||
-	process.env.IS_DEFAULT_BRANCH === undefined
+	// process.env.IS_DEFAULT_BRANCH === 'true' || // TODO: Disabled until we can fix the cache image generation
```

Comment:
We should have removed this -- it'll make the build faster on trunk because it won't persist things any more.